src/scvi/data/_download.py

```diff
@@ -1,56 +1,91 @@
 import logging
 import os
-import urllib
+from pathlib import Path
 
-import numpy as np
+logger = logging.getLogger(__name__)
 
-from scvi.utils import track
 
-logger = logging.getLogger(__name__)
+def _download_scanpy_style(url: str, path: Path):
+    try:
+        import ipywidgets  # noqa: F401
+        from tqdm.auto import tqdm
+    except ImportError:
+        from tqdm import tqdm
```

Comment:
What’s this test about? To support old versions can we just bump the requirement?

Reply:
We should remove it as in the other function below we just import from auto. Can you check that the requirement is correct and we always install a tqdm version that has auto?